src/material-experimental/mdc-card/card.ts

```diff
@@ -25,13 +25,16 @@ import {
   selector: 'mat-card',
   templateUrl: 'card.html',
   styleUrls: ['card.css'],
-  host: {'class': 'mat-mdc-card mdc-card'},
+  host: {'class': 'mat-mdc-card mdc-card',
+  '[class.mdc-card--outlined]': 'appearance == "outline"'},
```

Comment:
The formatting here is a little weird. Consider breaking it up into multiple lines.